src/GeneralStateTestsFiller/stEWASMTests/callNoDataNoValueFiller.yml

```diff
@@ -105,7 +104,6 @@ callNoDataNoValue:
             (set_local $memStart (i32.const 32))
 
             ;; init pointers
-            (set_local $ptrStorageKey1 (get_local $memStart))
```

Comment:
I think in this case you weren't supposed to remove this but the `i32.store` line a few lines down?